styling of movie page

Bekhalm opened the pull request:
Styled the movies on the movies page http://localhost:3000/api/movies.

Jorlindstrom commented:
Jag har jämfört din styling med hur filmstaden har stylat sin filmsida för att få en referens. se bilder.

**Titel**
Jag föreslår att namnet under filmerna borde vara en ljusare färg och gärna mindre typsnittsstorlek.

**Desktop:**
På desktopsstylingen. Storleken på filmbilderna känns för stora på min skärm enligt min smak. Jag är osäker om man ska boxa in filmerna i en maxstorlek eller om de ska få flöda fritt, hmm jag är osäker där. Jag föreslår iallafall att det sätta en marginal på var sida om skärmen.

**Läsplatta:**
Här skulle du kunna sätta 4 filmer i bredd med en liten marginal på var sida om skärmen. det känns liksom bättre att se ett slut innan skärmen upphör. annars ligger man vaken om natten och undrar om det egentligen finns något mer där ute=)

**Mobil:**
Här föreslår jag att sätta två filmer bredvid varandra på varje rad så att man slipper scrolla så långt för att få se den sista filmen, också här behövs en liten marginal på var sida om skärmen.

<img width="457" alt="Skärmavbild 2025-05-15 kl  12 09 46" src="https://github.com/user-attachments/assets/f39fa596-7d82-4cfa-9e5f-8756a1b8215f" />
<img width="428" alt="Skärmavbild 2025-05-15 kl  12 10 07" src="https://github.com/user-attachments/assets/549882d2-d6be-4f6a-8b7c-543a10d33d70" />
<img width="593" alt="Skärmavbild 2025-05-15 kl  12 11 20" src="https://github.com/user-attachments/assets/386087e2-8c9e-4a66-8286-e6c74ade04d9" />
<img width="596" alt="Skärmavbild 2025-05-15 kl  12 11 37" src="https://github.com/user-attachments/assets/fc3fe3f1-0214-44b7-a020-32edbb70976d" />
<img width="1443" alt="Skärmavbild 2025-05-15 kl  12 08 09" src="https://github.com/user-attachments/assets/00ac45dc-2455-4a38-b5fd-405da5c7d610" />
<img width="1455" alt="Skärmavbild 2025-05-15 kl  12 07 34" src="https://github.com/user-attachments/assets/0f026c50-6bf9-456b-bb0d-d7d573feab0a" />

Jorlindstrom reviewed (approved):
I have compared your styling with how Filmstaden has styled its film page to get a reference. see pictures.

**Title**
I suggest that the name under the films should be a lighter color and preferably a smaller font size.

**Desktop:**
On the desktop styling. The size of the film images feels too large on my screen for my taste. I am unsure whether to box the films into a maximum size or whether they should be allowed to flow freely, hmm I am unsure there. I suggest at least putting a margin on each side of the screen.

**Tablet:**
Here you could put 4 films in width with a small margin on each side of the screen. It feels better to see an end before the screen ends. Otherwise you lie awake at night wondering if there is really something more out there =)

**Mobile:**
Here I suggest putting two films next to each other on each line so that you don't have to scroll so far to see the last film, here too a small margin is needed on each side of the screen.

<img width="457" alt="Skärmavbild 2025-05-15 kl  12 09 46" src="https://github.com/user-attachments/assets/f39fa596-7d82-4cfa-9e5f-8756a1b8215f" />
<img width="428" alt="Skärmavbild 2025-05-15 kl  12 10 07" src="https://github.com/user-attachments/assets/549882d2-d6be-4f6a-8b7c-543a10d33d70" />
<img width="593" alt="Skärmavbild 2025-05-15 kl  12 11 20" src="https://github.com/user-attachments/assets/386087e2-8c9e-4a66-8286-e6c74ade04d9" />
<img width="596" alt="Skärmavbild 2025-05-15 kl  12 11 37" src="https://github.com/user-attachments/assets/fc3fe3f1-0214-44b7-a020-32edbb70976d" />
<img width="1443" alt="Skärmavbild 2025-05-15 kl  12 08 09" src="https://github.com/user-attachments/assets/00ac45dc-2455-4a38-b5fd-405da5c7d610" />
<img width="1455" alt="Skärmavbild 2025-05-15 kl  12 07 34" src="https://github.com/user-attachments/assets/0f026c50-6bf9-456b-bb0d-d7d573feab0a" />

nilsishome commented on src/app/movies/moviesPage.style.ts:
```diff
@@ -0,0 +1,54 @@
+};
+
+export const title: SxProps<Theme> = {
+  fontFamily: "initial",
```
Maybe remove this property?

nilsishome commented on src/app/movies/moviesPage.style.ts:
```diff
@@ -0,0 +1,54 @@
+  textAlign: 'center',
+  fontWeight: 500,
+  fontSize: 20,
+  color: 'black',
```
Maybe remove this color property as well... then it will take our default text color value.

nilsishome replied:
I will change the default text color in my PR, so the movie titles will get a lighter color, as Jörgen wrote in his review.

Bekhalm commented:
Thanks for the feedback!
I changed the color of the movie names to the same yellow we use throughout the site. I also adjusted the gap between the movies and made it responsive to different screen sizes!

nilsishome reviewed (approved):
Great job with this feature you've implemented and the fixes, Bekhal! I'm approving now :+1:
